CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
 **Fixed**
 
+* Cloud only: in OKE authentication, use crypto.randomBytes instead of
```

Comment:
this isn't user-visible is it? Even if so, it's for a feature that hasn't yet been visible so I don't see the need for CHANGELOG entry

Reply:
OK, I removed changelog changes.